tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/SearchModelTests.cs

```diff
@@ -24,22 +24,16 @@ public class SearchModelTests
 
     public SearchModelTests()
     {
-        Mock<ITrustService> mockTrustService = new();
-        _mockTrustSearch = new Mock<ITrustSearch>();
+        ITrustService mockTrustService = Substitute.For<ITrustService>();
+        _mockTrustSearch = Substitute.For<ITrustSearch>();
+        
+        mockTrustService.GetTrustSummaryAsync(_fakeTrust.Uid).Returns(_fakeTrust);
+        
+        _mockTrustSearch.SearchAsync(SearchTermThatMatchesAllFakeTrusts, Arg.Any<int>())
+            .ReturnsForAnyArgs(new PaginatedList<TrustSearchEntry>(_fakeTrusts, _fakeTrusts.Length, 1, 1));
```

Comment:
```suggestion
            .Returns(new PaginatedList<TrustSearchEntry>(_fakeTrusts, _fakeTrusts.Length, 1, 1));
```
We should use `Returns` rather than `ReturnsForAnyArgs` because we only want to return this if given `SearchTermThatMatchesAllFakeTrusts`. 

Doing this would also mean the other additional changes below aren't needed